sigstore/root-signing
# snapshot-timestamp for asraa

Signed-off-by: Asra Ali <asraa@google.com>



#### Summary


TO VERIFY:
```
REPO=$(pwd)/ceremony/2022-07-12  ./scripts/verify.sh 320
```

You should see
`timestamp.json` and `snapshot.json` validly signed under staged metadata, with versions 38.

#### Release Note


#### Documentation
